api/src/services/authentication.ts

```diff
@@ -171,7 +171,6 @@ export class AuthenticationService {
 			'auth.jwt',
 			tokenPayload,
 			{
-				status: 'pending',
```

Comment:
I believe it's to indicate the status of the login action, which at this point in the flow is still pending. Lets leave it in, otherwise it's a (small) breaking change 👍🏻